### ISSUE TITLE
3rd party coverage check tool shows wrong coverage stats

### ISSUE THREAD

**Issue by chrisleekr** [d0x2f/CloverMerge#4]
I am using `rregeer/phpunit-coverage-check` to check coverage threshold.

It seems `phpunit-coverage-check` tool only getting the coverage of one of xml files, not combined coverage.

```shell
$ ./vendor/bin/clover-merge -m exclusive -o build/logs/clover_merged.xml build/logs/clover_kahlan_functional.xml build/logs/clover_kahlan_unit.xml
Files Discovered: 85
Final Coverage: 1049/1479 (70.93%)

$ ./vendor/bin/coverage-check build/logs/clover_merged.xml 100 
Total code coverage is 49.76%, which is below the accepted 100%
```

Is it possible to let 3rd party tool can read combined coverage ?

**Comment by d0x2f** [comment]
This seems to be caused by the coverage-check tool checking the "metrics" XML node which isn't modified by CloverMerge.

I think the metrics should be updated and will make a patch for it.

**Comment by d0x2f** [comment]
This issue is resolved by a98d95fb16e0c4f20ae43d1b3c51852bf3d7010c

I've made a new release 'v1.0.4' including the change.
